File: problemas/iniciante/1002/1002.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        InputStreamReader ir = new InputStreamReader(System.in);
        BufferedReader in = new BufferedReader(ir);
        final double pi = 3.14159;
        
        double raio = Double.parseDouble(in.readLine());
        double area = pi * raio * raio;

        System.out.printf("A=%.4f\n", area);
    }
}

File: problemas/iniciante/1002/1002.cs
using System;

class HelloWorld {
    static void Main(string[] args) {
        const double pi = 3.14159;
        double raio = double.Parse(Console.ReadLine());

        double area = pi * raio * raio;

        Console.WriteLine($"A={area:0.0000}");
    }
}